static/app/views/performance/newTraceDetails/traceDrawer/details/styles.tsx

```diff
@@ -713,6 +728,143 @@ function DropdownMenuWithPortal(props: DropdownMenuProps) {
   );
 }
 
+type KeyValueActionProps = {
+  rowKey: string;
+  rowValue: React.ReactNode;
+  kind?: TraceDrawerActionValueKind;
+};
+
+function KeyValueAction({
+  rowKey,
+  rowValue,
+  kind = TraceDrawerActionValueKind.SENTRY_TAG,
+}: KeyValueActionProps) {
+  const location = useLocation();
+  const organization = useOrganization();
+  const hasNewTraceUi = useHasTraceNewUi();
+  const [isVisible, setIsVisible] = useState(false);
+
+  if (
+    !hasNewTraceUi ||
+    !defined(rowValue) ||
+    !defined(rowKey) ||
+    !['string', 'number'].includes(typeof rowValue)
+  ) {
+    return null;
+  }
+
+  // We assume that tags, measurements and additional data (span.data) are dynamic lists of searchable keys in explore.
+  // Any other key must exist in the static list of sentry tags to be deemed searchable.
+  const searchableSentryTags = [...SentryStringTags, ...SentryNumberTags];
```

Comment:
Creating a new array here just to check includes not great. You're probably better off just doing 2 `includes` or check the appropriate list based on the type